pkg/sql/catalog/accessor.go

```diff
@@ -40,11 +40,11 @@ type Accessor interface {
 		ctx context.Context, txn *kv.Txn, dbName string, flags tree.DatabaseLookupFlags,
 	) (DatabaseDescriptor, error)
 
-	// GetSchema returns true and a ResolvedSchema object if the target schema
+	// GetSchema returns true and a SchemaDescriptor object if the target schema
```

Comment:
Function name in docstring doesn't match declaration. Strange that the linter doesn't complain. I filed an issue: https://github.com/cockroachdb/cockroach/issues/65459